src/Native/build-native.cmd

```diff
@@ -9,6 +9,8 @@ set __rootDir=%~dp0..\..
 set __CMakeBinDir=""
 set __IntermediatesDir=""
 set __BuildArch=x64
+:: TODO: This should default to 'netcoreapp' whenever the managed assemblies do the same.
```

Comment:
You can actually change the managed side to netcoreapp today. I already did with my recent PR https://github.com/dotnet/corefx/pull/14550.